src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -2955,6 +2956,20 @@ cdef class SubsetInput:
             self._hb_input, face._hb_face, hb_tag, value
         )
 
+    def override_name_table(self, name_id: OTNameIdPredefined | int, platform_id: int, encoding_id: int, language_id: int, name_str: str | None) -> bool:
+        cdef const char* c_name
+        cdef Py_ssize_t str_len
+        if name_str is None:
+            c_name = NULL
+            str_len = -1
+        else:
+            c_name = PyUnicode_AsUTF8AndSize(name_str, &str_len)
+            if c_name == NULL:
+                raise MemoryError()
+        return hb_subset_input_override_name_table(
+            self._hb_input, name_id, platform_id, encoding_id, language_id, c_name, str_len
+        )
```

Comment:
Linux distributions builds want to link to system HarfBuzz which is typically built without experimental APIs, so this call should be wrapped in `HB_EXPERIMENTAL_API` and raise a `NotImplementedError` otherwise.